src/main/webapp/i18n/de/lecture.json

```
@@ -100,7 +100,14 @@
             "pdfPreview": {
                 "title": "Anhang",
                 "attachmentIDError": "Ungültiger Anhang oder ungültige Anhangs-ID.",
-                "attachmentUnitIDError": "Ungültige Dateieinheit oder ungültige Dateieinheits-ID."
+                "attachmentUnitIDError": "Ungültige Dateieinheit oder ungültige Dateieinheits-ID.",
+                "removePageButton": "Entferne Seiten",
+                "appendFileButton": "Anhänge Datei",
```

I would rephrase this to `Datei anhängen`
```suggestion
                "appendFileButton": "Datei anhängen",
```

---

Thanks, changed it :)